src/source/PeerConnection/PeerConnection.c

```diff
@@ -876,6 +933,8 @@ PVOID resolveStunIceServerIp(PVOID args)
     }
     releaseHoldOnInstance(pWebRtcClientContext);
     DLOGD("Exiting from stun server IP resolution thread");
+
```

Comment:
is the log required?

Reply:
It looks like this was added for the Debug level, whereas ENTERS and LEAVES are streaming level. I will keep the log there as it shouldn't cause any issues.